keps/sig-auth/4412-projected-service-account-tokens-for-kubelet-image-credential-providers/README.md

```diff
@@ -933,7 +943,7 @@ Steps to disable the feature:
 3. Restart the kubelet.
 
 These steps need to be performed on all nodes in the cluster.
-After restarting the kubelet on all nodes, remove the audiences used by kubelet from the KAS `--allowed-kubelet-audiences` flag.
+After restarting the kubelet on all nodes, remove the allowed audiences for which the kubelet is allowed to generate service account tokens for image pulls in KAS by removing the previous `ClusterRole` or `Role` with the `request-serviceaccounts-token-audience` verb.
```

Comment:
Am I misremembering or is this a cluster scoped check?  Basically, can you use a Role?

Reply:
Re-read the earlier parts of the KEP to remember the SAR includes namespace so Role works.  Just update this to mention that the binding needs to also be deleted.